addons/web_unsplash/controllers/main.py

```diff
@@ -81,7 +81,7 @@ def slugify(s):
         for key, value in unsplashurls.items():
             url = value.get('url')
             try:
-                if not url.startswith('https://images.unsplash.com/') and not request.env.registry.in_test_mode():
+                if not url.startswith(('https://images.unsplash.com/', 'https://plus.unsplash.com/')) and not request.env.registry.in_test_mode():
```

Comment:
Ideally avoid such long line, do something like:
```suggestion
                if (
                    not url.startswith(('https://images.unsplash.com/', 'https://plus.unsplash.com/'))
                    and not request.env.registry.in_test_mode()
                ):
```
(Using `()` is IMHO way cleaner/easier to read than using `\`

It's fine as is, let's merge it, but keep it in mind for next PRs :)